### DISCUSSION TITLE
Server modding support - remove lag / increase player limit

### DISCUSSION THREAD

**Discussion by kkoshelev** [mbround18/valheim-docker#157] [Ideas]
Basically patch the server as described - https://www.reddit.com/r/valheim/comments/lkea78/valheim_dedicated_server_lag/
There is also a way to increase player limit above 10 - https://www.nexusmods.com/valheim/mods/4 or https://umod.org/games/valheim/guide

This is nice to have -> Remove Item teleport prevention from all items

**Comment by mbround18** [comment]
I have done this for my server, as my modding disclaimer, I give you the tools to run a modded server but you have to install the mods yourself <3 if you need assistance on installing the mods please let me know. However, because of the complexity and the infancy of the Valheim modding community currently; this tool will not support installing a mod manager or various mods through Odin for the time being. 


This can change in the future but only once a dedicated modding tool is settled on and or the Valheim developers officially support modding. Rather than it being muddy waters at the moment.
